scanner/image/scanner/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
+
+LABEL name="scanner" \
+      vendor="StackRox" \
+      maintainer="support@stackrox.com" \
+      summary="Static vulnerability scanner for the StackRox Security Platform" \
+      description="This image supports static vulnerability scanning for the StackRox Security Platform."
+
+SHELL ["/bin/sh", "-o", "pipefail", "-c"]
+
+COPY ../../../image/static-bin/import-additional-cas ../../../image/static-bin/restore-all-dir-contents ../../../image/static-bin/save-dir-contents /
```

Comment:
Hmmm. This is pretty odd for a dockerfile to be going backwards in directory..., actually how is this even possible as docker build describes the build scope...

Reply:
yeah it doesn't work. I'm in the process of updating this PR, but I'm waiting on a few other PRs to be merged first :)